src/models/github-server-app.ts

```diff
@@ -161,19 +177,18 @@ GitHubServerApp.init({
 		type: DataTypes.STRING,
 		allowNull: false
 	},
-	secrets: encrypted.vault("secrets"),
-	gitHubClientSecret: encrypted.field("gitHubClientSecret", {
-		type: DataTypes.STRING,
+	gitHubClientSecret: {
+		type: DataTypes.TEXT,
```

Comment:
Might need some discussion here, but I read some article about `TEXT` is not worse than `STRING` anyway as underhood impl same

Reply:
if it's the same implementation, I'd suggest we stick with `STRING` as it makes more sense from a code perspective

Reply:
Oh I mean same underline impl, but the constraints are still differents (https://sequelize.org/v5/manual/data-types.html)
STRING: varchar(255) and TEXT no constraint but the db storage engine. (1gb?)

Reply:
ah right